src/NServiceBus.Transport.Bridge/NServiceBus.Transport.Bridge.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="8.0.0-rc.1" />
+    <PackageReference Include="NServiceBus" Version="[8.0.0-rc.1, 9.0.0)" />
```

Comment:
@bording @DavidBoike As this is downstream it seems this must be range. Is this a correct conversion?

Reply:
Yes, probably, but I was under the impression that a dedicated task force was managing this package.

Reply:
@DavidBoike it was, I asked @dvdstelt and he ack'd that it should be a range.